code/game/objects/items/storage/boxes.dm

```diff
@@ -1259,3 +1259,12 @@
 	var/obj/item/stack/sheet/cardboard/I = new(user.drop_location())
 	qdel(src)
 	user.put_in_hands(I)
+
+/obj/item/storage/box/marshmallow
+	name = "box of marshmallows"
+	desc = "A box of marshmallows."
+	icon_state = "marshmallowbox"

```

Comment:
this should be an `illustration` (those little overlays added to boxes to distinguish one another)